tests/utils.py

```diff
@@ -66,3 +67,27 @@ def assert_graph(graph, expected):
     elines = sorted(elines)
 
     assert elines == lines, (lines, elines)
+
+
+class AsyncClosing(object):
+
+    def __init__(self, awaitable):
+        self.awaitable = awaitable
+
+    async def __aenter__(self):
+        self.closable = await self.awaitable
+        return object
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
+        await self.closable.stop()
+
+
+def get_free_tcp_port():
```

Comment:
The deploy command currently allocates a random number. So maybe even move this to inmanta.util so deploy can use this as well.